### PR TITLE
docs: fix broken links in v1.4.3 documentation, other issues in #117

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**

documentation bug

**Which issue does this PR fix**:

Partial fix for issue #117 

**What does this PR do / Why do we need it**:

The initial commit is a scrub of all broken links, changed to valid links.
The second commit addresses the specific links in issue #117 that were not fixed in the first commit.
Some minor formatting fixes are also included.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/articles/authn-authz.md [hunk 1]
```diff
@@ -206,44 +206,46 @@ Use the `accessControl` attribute in the configuration file to define a set of i
 "http": {
 ...
   "accessControl": {
-    "**": {
```
Comment (rchincha) [inline docs/articles/authn-authz.md:209]: This doesn't look right.
In v1.4.3, the policy looks like this.
https://github.com/project-zot/zot/blob/v1.4.3/examples/config-policy.json#L16

Comment (rchincha) [reply docs/articles/authn-authz.md:209]: In your latest "What's new" document, we should also call out that the policy configuration has changed as your are pointing out here.

Comment (mbshields) [reply docs/articles/authn-authz.md:209]: This bug issue (https://github.com/project-zot/zot/issues/1673) says that the "repositories" attribute must be under the "accessControl" attribute.  Is this only for 2.0 and not for 1.4.3?

Comment (rchincha) [reply docs/articles/authn-authz.md:209]: Correct, always best to look under examples/ for **that** version

Comment (rchincha) [reply docs/articles/authn-authz.md:209]: We should actually call this out in a "NOTE" topic under what's new

Comment (mbshields) [reply docs/articles/authn-authz.md]: Removed 'repositories' under 'accessControl' -- not needed for v1.4.
Will add note about this change in What's New for v2.0.

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**